### DISCUSSION TITLE
Increase Raid Loot and Player Motivation

### DISCUSSION THREAD

**Discussion by Daniel7Ionescu** [chromiecraft/chromiecraft#3291] [Suggestions]
Increase the number of items raid bosses can drop by 1 or 2. It often happens that from 1 raid run we only use a handful of boss drops, not because we are over-geared but because most items are worse than pre-raid bis.  
  One issue that we are dealing with, is players are getting bored and stop logging on.  They have a bunch of raiding alts and just want to get them geared, as raiding is the only main activity on this server.
   Increasing the number of items dropped would make raiding more exciting and stop loosing some of the players.

Aquanox <Horizon> we are currently fielding 2 raid teams.

**Comment by 55Honey** [comment]
Hello b3zna,
Thanks for your suggestion.
I dont really see how giving more items and saturating the players demands even quicker can lead to longer term motivation?
Unless i misunderstood, this should have the exact opposite results.

**Comment by Daniel7Ionescu** [comment]
It all boils down to character progression and getting loot, thats what most players care about.
  In these raids there are no tier tokens, a good deal of the items from the bosses just aren't useful. Most of the times drops from bosses are disenchanted. Most of the players want to equip their character with best in slot for the current tier and be ready for the next one. Others have a bunch of alts that they want to gear up. 
  When these dedicated players get bored or burned out and move on to other servers it affects the whole raid roster and then more people lose interest and the raid team doesn't have enough players.
  Sadly the player pool has dropped drastically since MC has opened for various reasons, so we can't fill up the raid teams.
  In TBC this is not such a big issue because of Tier Tokens, Heroic dungeons and Reputation grind that gives great items, so players have more avenues of activity during a week /  raid reset to progress their character.
